packages/components/src/2016-12-01/button-state.test.js

```diff
@@ -16,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 */
-const chai = require('chai');
+const chai = require('chai'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I Agree - [import/no-extraneous-dependencies](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md) has a way to avoid this though so we should just configure it to ignore:

```
When using an array of globs, the setting will be activated if the name of the file being linted matches a single glob in the array.
```